src/Platform/Microsoft.Testing.Platform/Configurations/ConfigurationExtensions.cs

```diff
@@ -35,6 +35,7 @@ public static string GetCurrentWorkingDirectory(this IConfiguration configuratio
     /// </summary>
     /// <param name="configuration">The configuration.</param>
     /// <returns>The test host working directory.</returns>
+    // TODO: This is only used in unit tests. Should that be removed (maybe in v2)?
```

Comment:
in that case deprecate it?

Reply:
Marked as obsolete and cleaned up usages.